src/bot.ts

```diff
@@ -141,9 +142,14 @@ export function createBot(config: BotConfig) {
       return path.resolve(config.dataDir, `${content}.mp3`);
     }
     if (config.myInstantsEnabled && content.startsWith("instant")) {
-      const sound = encodeURIComponent(content.substr("instant".length).trim().substr(0, 256))
+      const sound = encodeURIComponent(content.slice("instant".length).trim().slice(0, 256))
       return `https://www.myinstants.com/media/sounds/${sound}.mp3`;
     }
+    if (config.wikiaCDNEnabled && content.startsWith("wikiacdn")) {
+      const link = content.slice("wikiacdn".length).trim().slice(0, 256);
```

Comment:
@alimg what was the significance of this 256?

Reply:
Nothing particular about 256. It's just for sake of preventing bad user provided input in the external call we gonna make as myinstants links are usually quite small.